composer.json

```diff
@@ -25,7 +25,8 @@
     "autoload": {
         "psr-4": {
             "Aura\\Filter\\": "src/"
-        }
+        },
+        "files": ["iso8859_1_to_utf8.php"]
```

Comment:
This should be `utf8_to_iso8859_1.php`.
Because `utf8_decode()` converts a string from UTF-8 to ISO-8859-1, replacing invalid or unrepresentable characters.
https://www.php.net/manual/en/function.utf8-decode.php